README.md

```diff
@@ -72,6 +74,32 @@ spec:
   - start with an alphanumeric character
   - end with an alphanumeric character
 
+### Custom Data Types
+
+#### duration-shorthand
+
+The duration shorthand is specified as a single–word string (no whitespaces) consisting
+of a positive integer `number` followed by a case–sensitive single–character `postfix`.
+
+Allowed postfixes are:
+
+- *m* – minutes
+- *h* – hours
+- *d* – days
+- *w* – weeks
+- *M* – months
+- *Q* – quarters
+- *Y* – years
+
+Examples: `12h`, `4w`, `1M`, `1Q`, `365d`, `1Y`.
+
+This specification does not put requirements on how (or whether) to implement each
+postfix, therefore implementers are free to pick an implementation that best suits
+their environments.
```

Comment:
Do you  mean the implementor can decide how many days there in a year or week?

Reply:
Yes. A typical week has 168 hours. In countries with daylight savings time there are also weeks that are 167 and 169 hours long. Same with days – most are 24h, some are 23 and 25h. Years are typically 365d except when they're 366d. How do you define a rolling month? Or a rolling quarter? (And maybe you just don't need those in your implementation, so you'll skip having to think about it.)

The idea is that for now we tell implementers to figure something out that best suits them. And if over time the community comes to a conclusion on how to best approach this, we can update the spec with a recommendation on how we think this should be handled.